unblob/handlers/executable/elf.py

```diff
@@ -93,11 +93,21 @@ def extract_initramfs(self, elf, inpath, outdir):
             initramfs_end = initramfs_size_offset - padding
             initramfs_start = initramfs_end - initramfs_size
 
-            carve_chunk_to_file(
-                outdir.joinpath("initramfs"),
-                file,
-                ValidChunk(start_offset=initramfs_start, end_offset=initramfs_end),
-            )
+            # initramfs can be turned off (https://www.linux.com/training-tutorials/kernel-newbie-corner-initrd-and-initramfs-whats/)
+            # in which case the above calculations most probably end up with bogus chunk offsets
+            if (
+                init_data.file_offset
+                <= initramfs_start
+                <= initramfs_end
```

Comment:
With this condition, `initramfs_start` and `initramfs_end` can be equal, which would lead to an `InvalidInputFormat` being triggered.

Reply:
Modified to `<`, so we are slightly more strict about it.